config/configuration.toml.in

```diff
@@ -10,6 +10,15 @@ machine_type = "@MACHINETYPE@"
 # trouble running pre-2.15 glibc
 kernel_params = "@KERNELPARAMS@"
 
+# Path to the firmware.
+# If you want that qemu uses the default firmware leave this option empty
+firmware = "@FIRMWAREPATH@"
+
+# Machine accelerators
+# space-separated list of machine accelerators to pass to the hypervisor.
+# For example, `machine_accelerators = "nosmm,nosmbus,nosata,nopit,static-prt,nofw"`
+machine_accelerators="@MACHINEACCELERATORS@"
```

Comment:
Thanks - but that should be "comma-separated" right?

----
Aside: it's worth pointing out that this is the first config option that takes a comma-separated list. I think the current approach (where minimal validation is performed on the list) is fine, but if a user sets the variable to something like:

```
machine_accelerators = "foo,,"
```

... I suspect qemu will get pretty unhappy (but I haven't tested it).

Reply:
ohh yep, sorry :(

Reply:
Fixed, thanks @jodh-intel